project/Build.scala

```diff
@@ -932,7 +985,15 @@ object Build {
     withCommonSettings(Bootstrapped).
     dependsOn(dottyCompiler(Bootstrapped) % "provided; compile->runtime; test->test").
     settings(commonBootstrappedSettings).
+    settings(stdlibBootstrappedCommonSettings).
     settings(
+      Compile / scalacOptions ++= {
+        Seq("-sourcepath", ((Compile/sourceManaged).value / "scala-library-src").toString)
```

Comment:
Looks like this was missing before, but you probably also want -sourcepath on Compile / scalaSource since you're putting files there.